### PR TITLE
Update cloud-cost-management.md

### PR DESCRIPTION
**Pull request by suryanarayanan-venkatesan**
Updated Release notes

# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the CODEOWNERS. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ x] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [ ] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: release-notes/cloud-cost-management.md [hunk 1]
```diff
@@ -20,20 +20,67 @@ Harness deploys changes to Harness SaaS clusters on a progressive basis. This me
...
+
+In this release, we have addressed this issue by adding rule filters specifically for cloud providers.
 
 ### Early access features
 
 This release does not include any early access features.
 
 ### Fixed issues
 
+* Previously, the incorrect error message was displayed when Delegate validation failed. (CCM-14963)
```
Comment (rajarshee-harness) [inline release-notes/cloud-cost-management.md:41]:
```suggestion
* Previously, the incorrect error message was displayed when Delegate validation failed in Kubernetes Quick Create. (CCM-14963)
```

File: release-notes/cloud-cost-management.md [hunk 2]
```diff
@@ -20,20 +20,67 @@ Harness deploys changes to Harness SaaS clusters on a progressive basis. This me
...
+<details>
+<summary>2023 releases</summary>
+
+#### November 13, 2023, version 81500
+
+##### New features and enhancements
+
+* Previously, the perspective list page lacked pagination for both the Card and List views, causing lag on the first render of perspectives list pages for some customers. (CCM-14018)
+
+  Now, each page is paginated, with a maximum of 20 perspectives displayed, effectively resolving the issue.
```
Comment (rishik30) [inline release-notes/cloud-cost-management.md:57]:
```suggestion
The problem has been fixed, the perspective list is now paginated, with a limit of 20 perspectives shown per page.
```

File: release-notes/cloud-cost-management.md [hunk 3]
```diff
@@ -20,20 +20,67 @@ Harness deploys changes to Harness SaaS clusters on a progressive basis. This me
...
+  Now, it will be enabled if Cluster data is present, ensuring users can view historical costs.
+
+  
+#### November 3, 2023, version 81400
+
+##### New features and enhancements
+
+* Previously, there was no option to export Recommendations as CSV files. (CCM-14274)
+
+  Now, we have added a new feature that enables users to export Recommendations as comma-separated values (CSV) files.
```
Comment (Additi) [inline release-notes/cloud-cost-management.md:76]: This works for all the recommendations. Only for asset governance recommendations details are not exposed, just the list view is exported. The support for the same will be added soon.

**Review by Additi — CHANGES_REQUESTED** [review]
Please update the details.

File: release-notes/cloud-cost-management.md (hunk 3 above)
Comment (suryanarayanan-venkatesan) [reply release-notes/cloud-cost-management.md:76]: Should i add this under `Note: ` section below the same line?

**Comment by aimurphy** [comment]
@suryanarayanan-venkatesan Could you please update the CCM release notes to use the [new format](https://harness.atlassian.net/wiki/spaces/DOCS/pages/21534966104/Release+Notes+Format+New+for+Oct+2023#Individual-release-note-pages-(aka-%E2%80%9Cmodule-pages%E2%80%9D))? You can also look at the [CI page](https://developer.harness.io/release-notes/continuous-integration) as an example.

Please also update the `date` in the metadata when you publish release notes. This date is what triggers the RSS feed updates.

**Comment by aimurphy** [comment]
@jyotiarora-harness @rdsoze Another one that has been open since last year. Please take action to merge or close.

**Comment by aimurphy** [comment]
Closing this PR. The release notes for these releases were published.

**Pull request closed without merging**